Gemfile

```diff
@@ -16,6 +16,7 @@ gem 'sass-rails'
 gem 'snappconfig'
 gem 'uglifier'
 gem 'whenever', require: false
+gem 'puma', '~> 3.7'
```

Comment:
So is `puma` already the default web server for rails and it's just not included in the gemfile?

Reply:
It's the default for new rails 5 apps and when this was created, it was in Rails 4 so didn't get it. Obviously `rails s` still works so it's fine.